.github/workflows/modular-arch-quality-gates.yml

```
@@ -0,0 +1,644 @@
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Detect changed modules
+        id: check-changes
+        run: |
+          # Simple change detection - get modules as space-separated string
+          CHANGED_MODULES=$(git diff --name-only ${{ github.event.pull_request.base.sha }}...${{ github.sha }} | grep "^frontend/packages/" | cut -d'/' -f3 | sort -u | tr '\n' ' ' | sed 's/ $//')
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)